tests.py

```diff
@@ -216,6 +216,60 @@ def test_create_or_update_create_time_catch_request_error(self, m):
                                                                "time",
                                                                "times"))
 
+    def test_create_or_update_time_with_junk_string_duration(self):
+        """Tests that TimeSync._create_time will fail if a string containing no
+        hours/minutes is entered"""
+        params = {
+            "duration": "junktime",
+            "project": "ganeti-web-manager",
+            "user": "example-user",
+            "activities": ["documenting"],
+            "notes": "Worked on docs",
+            "issue_uri": "https://github.com/",
+            "date_worked": "2014-04-17",
+        }
+       
+        self.assertEquals(self.ts._create_or_update(params, None,
+                                                              "time", "times"),
```

Comment:
Flake8 is going to complain that this is overindented for visual indent. `"time"` should line up with `params` on the line above it.